pkg/gadgets/trace/mount/types/types.go

```diff
@@ -15,25 +15,63 @@
 package types
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64   `json:"mntnsid,omitempty"`
-	Pid       uint32   `json:"pid,omitempty"`
-	Tid       uint32   `json:"tid,omitempty"`
-	Comm      string   `json:"comm,omitempty"`
-	Operation string   `json:"operation,omitempty"`
-	Retval    int      `json:"ret,omitempty"`
-	Latency   uint64   `json:"latency,omitempty"`
-	Fs        string   `json:"fs,omitempty"`
-	Source    string   `json:"source,omitempty"`
-	Target    string   `json:"target,omitempty"`
-	Data      string   `json:"data,omitempty"`
-	Flags     []string `json:"flags,omitempty"`
-	FlagsRaw  uint64   `json:"flagsRaw,omitempty"`
+	Comm      string   `json:"comm,omitempty" column:"comm,maxWidth:10"`
+	Pid       uint32   `json:"pid,omitempty" column:"pid,maxWidth:7"`
+	Tid       uint32   `json:"tid,omitempty" column:"tid,maxWidth:7"`
+	MountNsID uint64   `json:"mntnsid,omitempty" column:"mnt_ns,maxWidth:12"`
+	Operation string   `json:"operation,omitempty" column:"op,maxWidth:10,hide"`
+	Retval    int      `json:"ret,omitempty" column:"ret,width:3,fixed,hide"`
+	Latency   uint64   `json:"latency,omitempty" column:"latency,minWidth:3,hide"`
+	Fs        string   `json:"fs,omitempty" column:"fs,minWidth:3,hide"`
+	Source    string   `json:"source,omitempty" column:"src,minWidth:3,hide"`
+	Target    string   `json:"target,omitempty" column:"dst,minWidth:3,hide"`
+	Data      string   `json:"data,omitempty" column:"data,width:40,hide"`
+	Flags     []string `json:"flags,omitempty" column:"flagsArr,width:40,hide"`
+	FlagsRaw  uint64   `json:"flagsRaw,omitempty" column:"flagsRaw,width:40,hide"`
```

Comment:
We want to have only one column named `FLAG`:

```suggestion
	Flags     []string `json:"flags,omitempty" column:"flags,width:24,hide"`
	FlagsRaw  uint64   `json:"flagsRaw,omitempty"`
```